assets/scss/modules/_notifications.scss

```diff
@@ -157,3 +157,18 @@
     min-width: 100%;
   }
 }
+
+.highlight-message__indent {
+  margin: em(20) em(40) em(20) 0;
+}
+
+.highlight-message__icon-top {
+  background-position-y: em(20);
+}
+
+.highlight-message__dark {
+  background-color: $govuk-blue-colour;
+  color: #fff;
+  padding-bottom: em(30);
+}
+
```

Comment:
indent, icon-top and dark sound like variants of highlight-message, so the BEM format would be highlight-message--indent etc.